mesheryctl/internal/cli/root/relationships/relationship.go

```diff
@@ -0,0 +1,158 @@
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/system"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+	"github.com/manifoldco/promptui"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	outFormatFlag string
+
+	availableSubcommands = []*cobra.Command{ViewRelationshipsCmd}
+)
+
+var RelationshipCmd = &cobra.Command{
+	Use:   "relationships",
+	Short: "View list of relationships and detail of relationships",
+	Long:  "View list of relationships and detailed information of a specific relationship",
+	Example: `
+// To view list of components
+mesheryctl exp relationships list
+
+// To view a specific model
+mesheryctl exp relationships view [model-name]
+	`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		//Check prerequisite
+
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return utils.ErrLoadConfig(err)
+		}
+		err = utils.IsServerRunning(mctlCfg.GetBaseMesheryURL())
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+		ctx, err := mctlCfg.GetCurrentContext()
+		if err != nil {
+			utils.Log.Error(system.ErrGetCurrentContext(err))
+			return err
+		}
+		err = ctx.ValidateVersion()
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+		return nil
+	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.SystemModelSubError(fmt.Sprintf("'%s' is an invalid subcommand. Please provide required options from [view]. Use 'mesheryctl exp relationships --help' to display usage guide.\n", args[0]), "model"))
+		}
+		_, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return utils.ErrLoadConfig(err)
+		}
+
+		err = cmd.Usage()
+		if err != nil {
+			utils.Log.Debug(err)
+			return err
+		}
+		return nil
+	},
+}
+
+func init() {
+	ViewRelationshipsCmd.Flags().StringVarP(&outFormatFlag, "output-format", "o", "yaml", "(optional) format to display in [json| yaml]")
+
+	RelationshipCmd.AddCommand(availableSubcommands...)
+}
+
+// selectModelPrompt lets user to select a relation if relations are more than one
+func selectRelationshipPrompt(relationship []v1alpha1.RelationshipDefinition) v1alpha1.RelationshipDefinition {
+	relationshipArray := []v1alpha1.RelationshipDefinition{}
+	relationshipNames := []string{}
+
+	relationshipArray = append(relationshipArray, relationship...)
+
+	for _, relationship := range relationshipArray {
+		// here display Kind and EvaluationQuery as relationship name
+		relationshipName := fmt.Sprintf("kind: %s, EvaluationPolicy: %s, Type: %s,  SubType: %s", relationship.Kind, relationship.EvaluationQuery, relationship.Type, relationship.SubType)
```

Comment:
The golangci-lint run encountered an error. The specific error message indicates an issue with Sprintf related to `relationship.Type` being a func value, not called (govet).